lab-davidW/bst.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const BstNode = module.exports = function(val) {
+  this.val = val;
+  this.data = null;
+  this.parent = null;
+  this.left = null;
+  this.right = null;
+};
+
+//O(n^2)
+BstNode.fromArray = function(node, array) {
```

Comment:
So I believe you actually have to do a little more work in this function. You'll have to instantiate a root node, and then traverse through the left and ride side to assign values from the given array to each node per side.